### ISSUE TITLE
Adds abstract property to Article 

### ISSUE THREAD

**Issue by danbri** [schemaorg/schemaorg#276]
http://schema.org/Article

There is a proposal - see discussion in https://github.com/schemaorg/schemaorg/pull/228 - that we should add an 'abstract' property, rather than use 'description' for everything.

**Comment by csarven** [comment]
Consider slightly relaxing the semantics around "abstract" and not be specific to `schema:Article`. See also: https://github.com/schemaorg/schemaorg/pull/228#issuecomment-109749159

**Comment by danbri** [comment]
There is more discussion in the old pull request at #228, but let's continue here. 

From a quick look back at the discussion my sense is that the notion of a structured abstract is what adds complexity here. In a schema.org setting there are two obvious ways to do that: by having property values be markup (easier in RDFa, feasible in JSON-LD, tricky in Microdata), and by using more graph/triple-based entities.  

@sballesteros - can you summarize what's going on in https://github.com/scienceai/scholarly-article/issues/4 ? Did you go off the idea of a structured abstract?

**Comment by darobin** [comment]
Where the Scholarly Article ontology is concerned, we still need to support structure at the abstract level — the reason it isn't moving much is because we're hoping to just use schema.org and align :)

One use case that hasn't (to my knowledge) been discussed in these issues is that of capturing language information about an abstract. The reason for this is that it is common for articles written in non-English languages to have a same-language abstract and an English abstract (in some cases there can be more).

Where this runs into trouble is that even simple, single-paragraph abstracts need to contain some markup (emphasis, strong, possibly some maths). However, the RDF data model forbids `rdf:HTML` (and in fact anything but the dumbest literals) to sport language information. It's a rather daft limitation, but AFAICT it's not going away. There are various hacks that one can resort to, but none are really any good.

Because of this, even in cases when we are not facing abstracts with structured subsections, it would be most useful to have a `schema:abstract` (or broadened `schema:description`) containing some subtype of `CreativeWork` so that it could be given at the very least an `schema:isLanguage`.

I'll let @sballesteros speak to the broader subsections issue.

**Comment by sballesteros** [comment]
Still would love to be able to support structured abstracts (right now we are just using `schema:description` for the abstract).

This is the latest of what we were thinking:

- 1 new class: `Abstract` (subClassOf `CreativeWork`)
- Adding CreativeWork to the range of `description` or adding 1 new property: `hasAbstract` (domain: `CreativeWork`, range: `Abstract`)

Structured abstracts can then simply leverage the [`hasPart`](http://schema.org/hasPart) property of `CreativeWork`.

Example:

```js
{
  "@type": "ScholarlyArticle",
  "hasAbstract": { // or description
    "@id": "http://example.com/abstract",
    "@type": "Abstract",
    "hasPart": [
      {
        "@id": "http://example.com/abstract#Methods",
        "@type": [ "Abstract", "http://purl.org/spar/deo/Methods" ],
        "name": "Methods",
        "text": "We used ..."
      },
      {
        "@id": "http://example.com/abstract#Results",
        "@type": [ "Abstract",  "http://purl.org/spar/deo/Results" ],
        "name": "Results",
        "text": "We found ..."
      }
    ]
  }
}
```

 Not super happy with `hasAbstract` but it is to avoid `abstract` / `Abstract` which is discouraged by schema.org and seems to be a "pattern" (e.g [`hasDigitalDocumentPermission`](http://schema.org/hasDigitalDocumentPermission)  and [`DigitalDocumentPermission`](http://schema.org/DigitalDocumentPermission)).
 
Adding `CreativeWork` to the range of  `description` would remove the need for an `hasAbstract` property. I kind of like this solution (no metaphysical issue of what is the difference between an abstract and a description) but it is probably a bit too wild?

**Comment by darobin** [comment]
FWIW adding `CreativeWork` to the range of `description` is my favourite option here 🍦

**Comment by sballesteros** [comment]
I was looking at the [Google Knowledge graph API](https://developers.google.com/knowledge-graph/) and noticed that they use `detailedDescription` (see http://schema.googleapis.com/detailedDescription). That would work pretty well instead of bumping the range of description or adding a `hasAbstract` prop.

**Comment by darobin** [comment]
Does anyone have an objection to just reusing Google's `detailedDescription` here? At our end we've pretty much stretched reuse of `description` beyond usefulness or recognisability and are looking at just adopting that instead.

**Comment by danbri** [comment]
http://schema.org/Thing also has http://schema.org/disambiguatingDescription ...

**Comment by sballesteros** [comment]
Issue with disambiguatingDescription is that the range is `schema:Text` as opposed to `schema:CreativeWork` (goog:detailedDescription)

**Comment by danbri** [comment]
yeah, maybe we could fix that. I'd like to minimize use of goog-specific terms if we can. I think that one was added at Google for the Google KG API but I forget, maybe @vholland can advise?

**Comment by darobin** [comment]
My understanding is that `disambiguatingDescription` is completely different, it is meant to mark up "TV Series" in "Buffy (TV Series)", as in Wikipedia. I don't think it's a great idea to conflate the two. What this issue is about is more powerful descriptions, and since there's a `goog` extension the wild that seems to be a perfect match why not use it?

**Comment by thadguidry** [comment]
@darobin So in summary... your using description for a "short description" and would like a property for "much longer description" ?

**Comment by vholland** [comment]
As @darobin said, the intent of ```disambiguatingDescription``` was to add short phrases ala Wikipedia to disambiguate similarly named entities.

At Google, we have been experimenting with the property ```detailedDescription```, described as:

> An article which is a detailed description of the entity. The entity should be the primary focus of the article.

It's range is schema.org/Article, but I don't see why it could not be schema.org/CreativeWork instead.

**Comment by darobin** [comment]
@thadguidry It's not so much that as the fact that `description` is `Text`, whereas an abstract is more structured (sometimes a _lot_ more so) and makes sense as a `CreativeWork` (since it may have a title, different authors, languages, etc.). Based on this discussion overloading `description` for that feels like the wrong option.

**Comment by thadguidry** [comment]
@darobin OK, so since your need is really for dealing with Abstracts better and in having and providing structure FOR Abstracts...then the only way to solve that is with

:+1:  for adding a new class: Abstract (subClassOf CreativeWork)

**Comment by rvguha** [comment]
Can we have a more descriptive name (than Abstract)?

guha

**Comment by dbs** [comment]
If we're going to create a new type, let's call it what it is: StructuredAbstract.

**Comment by rvguha** [comment]
We should discuss that.

Another possibility is ArticleAbstract. Just don't want it to be confused
with the many Abstract X (such as Abstract Class). Just being super careful

guha

**Comment by Dataliberate** [comment]
Yes, *Abstract* is a dangerously generic word.

StructuredAbstract does describe the type of ’Thing’ it would be.

This definition <http://www.medilexicon.com/dictionary/327> kind of sums it
up:

Summary description of a published paper, *or other document,* in which
information about the study reported in the paper is set out in a
systematic, stylized form with separate paragraphs broken down under
headings such as aims, methods, main outcome measures, results,
conclusions. *[My insert]*


ArticleAbstract is another good candidate with a similar definition, unless
there are other abstracts of documents, that are not articles, that we
would want to encompass with this.

~Richard


Richard Wallis
Founder, Data Liberate
http://dataliberate.com
Linkedin: http://www.linkedin.com/in/richardwallis
Twitter: @rjw

**Comment by thadguidry** [comment]
I'd actually just prefer ... that we intend for the real reasoning behind it all... abstacting a CreativeWork (summarizing) ...so...

CreativeWorkAbstract

is a subtype and in the end, we are simplifying data about any CreativeWork.
@dbs (with your library hat on) what do you think ?

**Comment by philbarker** [comment]
No all abstracts are structured abstracts, so StructuredAbstract makes perfect sense to me.  Ordinary abstracts are just text.

@thadguidry as far as I can see the proposal is that it abstracts a ScholarlyArticle, not any CreativeWork. That seems the scope for which structured abstracts are a significant type of thing.

I quite like hasAbstract as a property of ScholarlyArticle (excepting the name) as it would cut out some ambiguity and graph hopping when looking for abstracts.

**Comment by RichardWallis** [comment]
Wishing to provide the focused functionality we are looking for, and providing as much generic usefulness at the same time how about:

New Type:
**StructuredAbstract**
subType of CreativeWork 
"_Summary description of a published paper, or other work, in which information about what is reported in the paper is set out in a systematic, stylised form with separate paragraphs broken down under headings such as aims, methods, main outcome measures, results, conclusions._"

New Property:
**hasStructuredAbstract**
"_Structured abstract of this work_"
domainIncludes: Article (_or possibly CreativeWork_)
rangeIncludes: StructuredAbstract, URL

New Property:
**abstractOf**
"_Work that is is the focus of this abstract_"
domainIncludes: StructuredAbstract
rangeIncludes: CreativeWork

The inverse property _abstractOf_ may seem a little superfluous, however it could be useful in certain situations such as for abstracting services.

**Comment by sballesteros** [comment]
So for the property, I kind of think that `detailedDescription` is a better fit that `hasStructuredAbstract` as it applies to more use cases than just abstracts. I would like to avoid having to think about things like: should i use `description` or `abstract` but just think in terms of do i just want to plug in the text (`description`) or additional metadata about the description as well (`detailedDescription`). `detailedDescription` with a broad range (`CreativeWork`) avoids that confusion while giving author the opportunity to use the right subclass of `CreativeWork` for the value.

With respect to the class (`StructuredAbstract`) my concern is that (at least in science), a structured abstract is an established concept: see https://www.nlm.nih.gov/bsd/policy/structured_abstracts.html

> A structured abstract is an abstract with distinct, labeled sections (e.g., Introduction, Methods, Results, Discussion) for rapid comprehension (see Figure 1)

So using `StructuredAbstract` for unlabeled abstracts (where we just want access to more properties from CreativeWork such as `inLanguage` etc.) might seem weird to some authors. I think that we could prevent that confusion with a different name (`CreativeWorkAbstract` or `ArticleAbstract`).

**Comment by thadguidry** [comment]
@RichardWallis @philbarker AH ! Now I see the disconnect between my thoughts and others.  Your treating Abstract as just a blob of text, perhaps less than 20 lines...like arXiv does: https://arxiv.org/help/prep#abstracts

I saw the need for more structure for an Abstract. Where some metadata is sometimes available for the section / paragraph of the Abstract.  But perhaps you think its not really needed. But I'm telling you, you'll want at least some kind of "conclusion" or "summary" property for any StructuredAbstract.  99% of them have one.   And why I was asking for more than a blob of text.

StructuredAbstract:section1 "Objectives"
StructuredAbstract:paragraph3 "Conclusions"

@RichardWallis I am ok with starting out with those 2 properties.  We'll probably need a 3rd for "summary" or "conclusion", etc. is my hunch. :)

**Comment by darobin** [comment]
@philbarker Actually, there are lots of abstracts that are not "structured abstracts" but are more than (or different from) text. At the very least abstracts, even the very simplest ones, almost always markup and a language. Since it is impossible to apply a language to `rdf:HTML` content, we need an object to apply `inLanguage`. Structured abstracts tend to reproduce the structure of what they're abstracting; it's a term of art and I believe it would be confusing to label all abstracts with the name of a specific kind of abstract. It's a bit like calling all mammals "elephants".

Graphical abstracts are also relatively common, they would have to be structured (you need to assemble text, titles, images) but they would not constitute what the scholarly world would recognise as a "structured abstract".

Note that there is also no requirement that one would have a `ScholarlyArticle` range, or that the abstract be somehow textual. For instance, standards have abstracts (though rarely structured in the scholarly sense). In video publications we have the case of a video abstract being a shorter version of the video (think trailer but without the voice from a hell dimension). Therefore `CreativeWork` is a sensible domain.

That's one reason I like `detailedDescription` as a name. It's a description, but you know, detailed.

**Comment by thadguidry** [comment]
@RichardWallis Looking at schema:Article... that actually fits fairly well for subtyping... it already has articleSection ! So we get Abstract sections for free :)

**Comment by RichardWallis** [comment]
@sballesteros I think we may be trying to satisfy two concerns here.

1. Some people are looking to markup Structured Abstracts as defined "_A structured abstract is an abstract with distinct, labeled sections..._" - see @thadguidry's ambitions for capturing even more markup about that structure.

2. Others, as you describe, are interpreting the term 'abstract' to mean a more detailed description than you would expect in ```description```.

It would be nice to try and solve both with the same proposal or perhaps this is a conflation that might be too confusing to solve [very] simply.

So evolving my previous suggestion based on comments:

New Type:
**WorkAbstract**
subType of CreativeWork 
"_Detailed or summary description of a CreativeWork (or subtype such as Article, Book, Video, etc.)_"

New Type:
**StructuredAbstract**
subType of WorkAbstract 
"_Summary description of a published paper, or other work, in which information about what is reported in the paper is set out in a systematic, stylised form with separate paragraphs broken down under headings such as aims, methods, main outcome measures, results, conclusions._"

New Property:
**abstract**
"_Structured abstract of this work_"
domainIncludes:  CreativeWork
rangeIncludes: WorkAbstract, URL

New Property:
**abstractOf**
"_Work that is the focus of this abstract_"
domainIncludes: WorkAbstract
rangeIncludes: CreativeWork

**Comment by philbarker** [comment]
@thadguidry @darobin  Agreed there are StructuredAbstracts, (what I think of as) ordinary abstracts that are just text, and others as yet not fully defined.

@darobin: 
> "I believe it would be confusing to label all abstracts with the name of a specific kind of abstract. It's a bit like calling all mammals "elephants"." 

--that's exactly what I was worried about. I am not worried about calling all elephants "mammals", so I would be happy to call the other as yet undefined abstracts anything from text through CreativeWork to Abstract

**Comment by philbarker** [comment]
@Dataliberate 
Having the definition of the property abstract mention "structured abstract" but the range not being StructuredAbstract seems odd.

**Comment by RichardWallis** [comment]
@philbarker true!

How's this tweak?:

New Property:
abstract
"Abstract or detailed description of this work"
domainIncludes: CreativeWork
rangeIncludes: WorkAbstract, URL, Text

**Comment by thadguidry** [comment]
@RichardWallis Ship it !

**Comment by csarven** [comment]
+1 to @RichardWallis 's suggestion https://github.com/schemaorg/schemaorg/issues/276#issuecomment-265490152 to include Text. I've been using schema:abstract with ^^rdf:HTML.

I think that the following would be sufficient to wrap up this issue:

domainIncludes: CreativeWork
rangeIncludes: URL, Text

with is practically https://github.com/schemaorg/schemaorg/issues/276#issuecomment-109749204

If the new class types can be well-defined (in another issue) and incubated, it'd make sense to introduce them through there. I kind of like the idea of adding WorkAbstract but only if Text is a possible range, and maybe if new properties that's interesting for an instance of WorkAbstract. Otherwise, I'm not sure if I actually see the benefit of introducing a new class which doesn't particularly make it more interesting/useful then CreativeWork.

[And for the love of the Web, can we avoid using the term *paper* when actually talking about *digital* articles on the Web?]

**Comment by danbri** [comment]
I've been about to raise the question of HTML content values, but it seems a shame to complicate as we have something approaching consensus here. Markup-valued properties are difficult or barely possible (need escaping?) in Microdata, but we could support them in RDFa and JSON-LD. Not sure whether we ought to have a HyperText/TextMarkup/HTMLText datatype alongside Text or some related idiom...

**Comment by csarven** [comment]
If the definition of schema:Text is **strictly** non-markup text, then I can see that different datatypes would be necessary for markup literals. I surely hope that's not the case with schema:Text right now. I'm treating it as both plain text as well as for markup (with a datatype accompanied) as needed. Which is also great to avoid additional datatypes for *text* in schema.org.

If schema:Text is only for non-markup text, does that exclude Markdown or some other convention for text? If so, why would any convention be different?

Some typographical rules can only be represented through a domain's own conventions. Ignoring that possibility would mean that the value of say schema:abstract or even schema:*Abstract's own properties will have to be strictly non-markup. I don't think that's even the case in practice e.g., authors do *emphasise* their text at least for human readers. This is not to say that all properties with Text is expected to have markup e.g., a person's name mostly likely doesn't include some form of markup. I assume that the current usage of schema:Text out there includes both plain literals as well as markup. Do you know?

**Comment by darobin** [comment]
This may be veering somewhat off-topic but I am increasingly operating under the assumption that _all_ text may be markup (with accompanying datatype of course, otherwise this is madness — and I would do the same if I used MD).

You mention the case of a `Person`'s name — it could for instance feature ruby annotations for pronunciation (though that's admittedly rare) or directionality information for people who have multi-script names (also rare, but I've heard of Arabic texts in which a person might use a latinised nickname with their Arabic family name). If you consider the `name` property more broadly, then you definitely have markup all over.

So far almost every time I've assumed that something would only ever be unmarked text I've been wrong. That Microdata cannot represent that is just another nail in that specific coffin. Having said that RDF is also maddeningly broken here since you can't attach language information to anything other than a plain text literal. I'm not sure how that got past the I18N people. I haven't seen any interest in W3C for fixing that, sadly.

**Comment by sballesteros** [comment]
If we agree that there is a use case to provide a `CreativeWork` as value for things that fall under the broad category of `schema:description` then I think that a property of the level of generality of `detailedDescription` is better suited than `abstract` as the use case is of interest for more creative works than just articles (Audio, Video, etc). It may be worth considering starting with that and then adding sub properties when needed (and ask ourselves if we really need an `abstract` subproperty should we already have access to `description` and `detailedDescription` ?). 

With that in mind, for the abstract use case we would have: 

```
"detailedDescription": {
  "@type": "WorkAbstract"
  ...
}
```
taking advantage of the new `WorkAbstract` class proposed by @RichardWallis .

And, for the knowledge graph API use case:

```
"detailedDescription": {
  "@type": "Article"
  ...
}
```

**Comment by csarven** [comment]
@darobin I shouldn't have generalised with 'mostly likely'. You are correct.

@sballesteros The idea of an 'abstract' or 'summary' of some resource is not novel and already exists in the wild. I think we agree that it differs from description, title/label of some resource. It sits somewhere in between. How people use abstract in practice may of course vary quite a bit. Having 'abstract' is not competing with a more complex and structured Abstract. I'd like to see both cases covered. And prefer not to hop around another resource to get to a simple abstract/summary.

**Comment by csarven** [comment]
Ping. Are we all still awake? :)

Focusing only on the core aspect of this issue ie adding abstract property, did we cover what we needed to cover? Do we agree that an abstract can have a Text, URL, and/or ExampleClass range?

IMO, a follow-up issue can address what's discussed here regarding structured abstracts: do we need a different (sub)property and class for structured/detailed abstracts?

The reason I prefer to separate these issues is because AFAIK we don't have a single schema for structured abstracts, so I'm hesitant to see that this issue covers everything all at once. I suspect that different research disciplines have their own structures, and different communities would have their own expectations going forward. Can we do this incrementally without over-specifying up front?

**Comment by danbri** [comment]
Any objections to adding "abstract" as a sub-property of "description". We can come back to structured text, markup etc separately.

**Comment by mfhepp** [comment]
Good idea!

**Comment by HughP** [comment]
@danbri Why not a sub property of "article"? is it because abstracts are broader than articles? Are we saying that a movie trailer is not an abstract of a movie (film)? I have never considered an abstract to be a type of description, though I guess it sorta does a descriptive job.

**Comment by csarven** [comment]
abstract subPropertyOf description sounds good to me. dcterms has it as well.

**Comment by danbri** [comment]
to @HughP 's point - "article" is not a property. A sub-property of one property is another that means the same thing and then something extra too. 

So we could say "abstract" is a property of a CreativeWork such as an Article, Book or other (typically written) work, which summarizes briefly the main content.

**Comment by danbri** [comment]
Ok, let's get this into our next release, for Sept 1st.

I had hoped to get it into this one ([3.9](https://schema.org/docs/releases.html#v3.9)) but felt duty bound, in my editorial capacity, to re-read this entire multi-year discussion to check we weren't missing anything. But maybe we can share that work? 

We got bogged down with structured abstracts previously. Can we live with a text field only, for now? @csarven, you asked also for URL values. What would they point to?

So, how about this:

* a new property, "/abstract", subPropertyOf "description".
* rangeIncludes: Text; domainIncludes: CreativeWork
* definition: "An abstract is a short description that summarizes a CreativeWork."

We should collect up the usecases that are not addressed by this, too

**Comment by csarven** [comment]
I can certainly live with Text for now. I've asked for URL as well because I figured that will make it possible for structured abstracts to be worked in later (through a separate issue). Are there any particular issues to add the URL later along with the notion of a structured abstract if/when that's ready?

+1 to the proposed property characteristics.

**Comment by danbri** [comment]
ok, I've committed a definition, and am republishing our editorial/drafting site on the webschemas domain... should be on https://webschemas.org/abstract in a few mins

**Comment by danbri** [comment]
@csarven we very often tweak the expected types associated with properties, so there's no need to anticipate all future developments at this moment. We prefer to do that than introduce lots of properties with overlapping meanings...

**Comment by fabricetheytaz** [comment]
Please, don't use reserved programming language keyword for label of type or property :pray:

**Comment by danbri** [comment]
there are too many programming languages for this to be feasible

**Comment by csarven** [comment]
Some implementation report for the record:

I've been using `schema:abstract` in some of the articles under https://csarven.ca/ since 2015.

dokieli is a clientside editor for decentralised article publishing, annotations and social interactions: https://dokie.li/ . Its UI allows users to embed statements with this property in their documents.

The W3C Linked Data Notifications spec: https://www.w3.org/TR/ldn/ also includes `schema:abstract` ;)

**Comment by coolharsh55** [comment]
I'm in the process of adding semantics to my publications list on website. `schema:abstract` would be more suitable there in addition to having a one-liner using `schema:description`. It would also be more suitable for longer publications on the web and scholarly articles. I support adding this to the vocabulary.

**Comment by dbs** [comment]
I've come around to being in favour of the addition of schema:abstract, but it needs some examples that incorporate schema:description as well so that people can clearly distinguish the purposes of the two properties.

Those examples could also show up under schema:description, of course, which would help lead people to the new schema:abstract property.

**Comment by coolharsh55** [comment]
> some examples that incorporate schema:description as well so that people can clearly distinguish the purposes of the two properties.

Regarding this article, `schema:description` could be "discussion on adding abstract property to schema.org" and `schema:abstract` could be: "This thread discusses the addition of an abstract property in addition to the existing description property. There is an existing PR (#228) which lists previous discussion on this topic. Current summary includes making abstract a sub-property of description and clarifying the differentiation between the two." In terms of semantics, `description` describes some artefact, and `abstract` provides an overview (or abstraction) of an artefact.

**Comment by dbs** [comment]
@coolharsh55 I understand the distinction, having been involved in the discussions since the PR, but most webmasters (or anyone who hasn't delved into this repository) will not. That's one of the main reasons for (and the value of) examples in schema.org : they help codify the usage of the classes and properties.

**Comment by RubenVerborgh** [comment]
More implementation reports: using it now for all [blog posts](https://ruben.verborgh.org/blog/), [articles](https://ruben.verborgh.org/articles/), and [publications](https://ruben.verborgh.org/publications/) on https://ruben.verborgh.org/. 338 occurrences in total.

**Comment by DamonHD** [comment]
In case other examples of use are valuable, I use abstract as part of a pyramid content hierarchy (headline, description, abstract) throughout Earth.Org.UK, eg see:

http://www.earth.org.uk/note-on-site-technicals-13.html

Aside: what's a good way to find out how many pages/sites are using various parts of the schema?  How weird am I to be doing this?  B^>

Rgds

Damon

**Comment by DamonHD** [comment]
If that's a question to me: I don't know.  Mine tend to run as long as a few sentences.  Search engines are probably not treating them much different than un-marked-up text at least for now is my guess.

**Comment by neliyoavi** [comment]
My privacy data has been stolen and published for ads
Help

**Comment by DamonHD** [comment]
So does 'closed'/'completed' mean that 'abstract' **IS** now becoming a fully-paid-up element of Article?

**Comment by gmackenz** [comment]
We are closing issues that have had no action/discussion in the past few years (as there are currently 339 current open issues from 2015 and later). If there is an issue you feel still needs resolving please summarize problem and potential fixes if any.

**Comment by DamonHD** [comment]
1) Is abstract going to remain a tentative/new child of Article as now?

2) Is abstract going to become a permanent child of Article?

3) Is abstract going to disallowed as a child of Article?

**Comment by fthobe** [comment]
@gmackenz 
[Abstract on schema.org](https://schema.org/abstract) is present on creativework. I think this can be closed.

**Comment by gmackenz** [comment]
No there seems to be some interest. From @DamonHD:

> So does 'closed'/'completed' mean that 'abstract' **IS** now becoming a fully-paid-up element of Article?

By inheritance yes, [Article](https://schema.org/Article) is a subtype of [CreativeWork](https://schema.org/CreativeWork) and [abstract](https://schema.org/abstract).

**Comment by fthobe** [comment]
@gmackenz @DamonHD 

Hey, 

What can we do here?

**Comment by gmackenz** [comment]
I will close this (again).
